docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -88,6 +88,37 @@ When I wait for presence of element by `$.data.testData` with `PT5S` polling int
 |When I issue a HTTP GET request for a resource with the URL '${requestUrl}'|
 ----
 
+=== Adding headers to HTTP request
+
+==== *_Info_*
+
+Adds request headers to request
```

Comment:
please include some link to docs about req headers

Reply:
too many `request` words in one sentence, please, make it concise